debian/rules

```diff
@@ -5,19 +5,23 @@
 
 
 # see FEATURE AREAS in dpkg-buildflags(1)
-#export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
 # see ENVIRONMENT in dpkg-buildflags(1)
 # package maintainers to append CFLAGS
 #export DEB_CFLAGS_MAINT_APPEND  = -Wall -pedantic
 # package maintainers to append LDFLAGS
-#export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
+export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
 
 export QT_SELECT := 5
 
 %:
 	dh $@ 
 
+override_dh_auto_configure:
+	# The existence of an empty .git directory triggers syncqt.
+	mkdir .git || true
+	qmake CONFIG+=packaging CONFIG-=debug CONFIG+=release
```

Comment:
@ZetaoYang please consider replacing this line with:

```makefile
override_dh_auto_configure
	# The existence of an empty .git directory triggers syncqt.
	mkdir .git || true
	dh_auto_configure -- CONFIG+=packaging CONFIG-=debug CONFIG+=release
```

so that Debian's options for qmake can be passed as well.

(FYI, Debian will use the following cmdline options for qmake:
`qmake -makefile "QMAKE_CFLAGS_RELEASE=-g -O2 -fdebug-prefix-map=<CURDIR>=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2" "QMAKE_CFLAGS_DEBUG=-g -O2 -fdebug-prefix-map=<CURDIR>=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2" "QMAKE_CXXFLAGS_RELEASE=-g -O2 -fdebug-prefix-map=<CURDIR>=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2" "QMAKE_CXXFLAGS_DEBUG=-g -O2 -fdebug-prefix-map=<CURDIR>=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2" "QMAKE_LFLAGS_RELEASE=-Wl,-z,relro -Wl,-z,now -Wl,--as-needed" "QMAKE_LFLAGS_DEBUG=-Wl,-z,relro -Wl,-z,now -Wl,--as-needed" QMAKE_STRIP=: PREFIX=/usr`
)

Reply:
@hosiet You're right.